package-lock.json

```diff
@@ -1,3264 +1,8 @@
 {
     "name": "nisaba",
     "version": "0.0.1",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
Interesting, what's your version of `npm`? I can update and regenerate it if that's the problem.

Reply:
```console
% npm --version
7.20.5
```
It isn't a huge deal, but this is going to always be modified when one or the other updates this file.  Maybe we should just delete it? :grimacing:

Reply:
Mine was 7.20.3, hm.
I don't mind deleting it. It's useful for some things but I'm not sure we're using any of them. My impression is that javascript projects resign themselves to having a diff in the package-lock.json that they essentially ignore. See [discussion and suggestions]: seems we should maybe use `npm ci` instead of `npm install`, which won't change the file?